lms/djangoapps/mobile_api/users/views.py

```diff
@@ -221,6 +221,7 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           * video_outline: The URI to get the list of all vides the user can
             access in the course.
           * id: The unique ID of the course.
+          * channel_id: A unique "clean" (alphanumeric with '_') ID of the course.
```

Comment:
``subscription_id`` sounds good to me. Are you planning to update mobile3 again? If so just let me and @hanningni so we can update the apps then. If not, we'll just go ahead and do it once you confirm.